src/flint/FlintTypes.jl

```diff
@@ -51,6 +51,11 @@ mutable struct ZZRingElem <: RingElem
     return z
   end
 
+  function ZZRingElem(::Val{:raw})
+    z = new(0)
+    return z
+  end
```

Comment:
This PR is not using this dangerous constructor, is it?

I think this should have at least a comment warning about its danger and clarifying what it might be useful for?

Reply:
There is no real danger attached. If you use this in large scale without a clue, than you'd be loosing memory, potentially.
But nothing will crash because of it